e2e/playwright/authentication/project.json

```diff
@@ -7,7 +7,7 @@
     "e2e": {
       "executor": "nx:run-commands",
       "options": {
-        "commands": ["npx playwright test --config=e2e/playwright/authentication/playwright.config.ts"]
+        "commands": ["playwright test --config=e2e/playwright/authentication/playwright.config.ts"]
```

Comment:
You can treat that comment as optional but if you think that may be helpful then we can probably simplify that command and reuse it too thanks to that we have similar folders structure for each e2e: 
1. You can add following script to package.json: 
"e2e": "playwright test --config=e2e/playwright/$npm_config_project/playwright.config.ts"

2. Then you can use that script in following way: 
npm run e2e --project=viewer

Thanks that it will inject "project" name into path. You can name script and parameter as any name you prefer